docs/migrations/00372.md

```diff
@@ -0,0 +1,53 @@
+# Purpose
+
+The purpose of this change was to converge on a standardized way of using change handlers.
+
+In Base UI we follow this event handler pattern:
+
+- custom handlers, or handler overrides with custom logic follow use the `onChange({event, value})` method signature,
```

Comment:
`with custom logic follow use the` => `with custom logic follow the` | `with custom logic use the`